test/acceptance/test_core_modules.js

```diff
@@ -0,0 +1,43 @@
+var assert = require('chai').assert,
+    c = require('../client.js');
+
+describe('Test core modules', function() {
+    this.timeout(5000);
+    before(function(done) {
+        client = new c(done);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [A constructor name should not start with a lowercase letter.](https://www.codacy.com/app/theprojectshop/Kassy/file/2531448132/issues/source?bid=3282987&fileBranchId=3283883#l7)